### PR TITLE
refactor: flatten constraint layout 

### PR DESCRIPTION
**Pull request by Gijuno**
in fragment_home_slide.xml

### CODE REVIEW

File: app/src/main/res/layout/fragment_home_slide.xml [hunk 1]
```diff
@@ -11,224 +11,191 @@
...
                 android:layout_height="wrap_content"
                 android:fontFamily="@font/noto_sans_kr_medium"
                 android:text="알고있는 정보를 공유하고 코인을 받을 수 있어요."
                 android:textColor="@color/color_gray"
-                android:textSize="14sp" />
+                android:textSize="14sp"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/ad_title_textview" />
 
         </LinearLayout>
```
Comment (nutyworks) [inline app/src/main/res/layout/fragment_home_slide.xml:146]: LinearLayout can be flattened with `app:layout_constraintVertical_bias` property.

**Review by nutyworks — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_home_slide.xml (hunk 1 above)
Comment (Gijuno) [reply app/src/main/res/layout/fragment_home_slide.xml:146]: You mean refactor 119 .. 122 line to layout_constraintVertical_bias?

Comment (nutyworks) [reply app/src/main/res/layout/fragment_home_slide.xml:146]: You can remove LinearLayout and use `app:layout_constraintVertical_bias` property on TextViews.

File: app/src/main/res/layout/fragment_home_slide.xml [hunk 2]
```diff
@@ -11,224 +11,185 @@
...
-                android:textSize="14sp" />
-
-        </LinearLayout>
+            android:fontFamily="@font/noto_sans_kr_medium"
+            android:text="알고있는 정보를 공유하고 코인을 받을 수 있어요."
+            android:textColor="@color/color_gray"
+            android:textSize="14sp"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/ad_title_textview"
+            app:layout_constraintVertical_bias="0" />
```
Comment (nutyworks) [inline app/src/main/res/layout/fragment_home_slide.xml:139]: Please constraint TextViews to icons, not the parent.

**Review by nutyworks — CHANGES_REQUESTED** [review]

**Review by nutyworks — APPROVED** [review]
LGTM 👍

**Pull request merged**